nornir/plugins/connections/netmiko.py

```diff
@@ -29,24 +29,22 @@ def open(
         hostname: str,
         username: str,
         password: str,
-        ssh_port: int,
-        network_api_port: int,
-        operating_system: str,
-        nos: str,
+        port: int,
```

Comment:
Same item of whether `port` can be None.

Reply:
There were a couple of other places with respect to Port where I had this same concern.

Reply:
Will try to dig them out.